pkg/registry/utils/inject/injecterror/error_supplier.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
```

Comment:
```suggestion
// Copyright (c) 2021 Doc.ai and/or its affiliates.
```
Other files in the `injecterror` package also have incorrect copyright holder info.

Reply:
These files all are actually copied from `networkservice/.../injecterror` so I want to leave original copyright.

Reply:
Ok, understandable